Dockerfile

```diff
@@ -223,7 +223,7 @@ COPY --from=builder /out/libexec/ /usr/local/libexec/
 ENV MAFFT_BINARIES=/usr/local/libexec
 
 # Ensure all container users can execute these programs
-RUN chmod a+rX /usr/local/bin/* /usr/local/libexec/*
+RUN chmod a+rx /usr/local/bin/* /usr/local/libexec/*
```

Comment:
The original use of `+X` was intentional as sometimes there are non-executable things in libexec (and more rarely bin) and so I wanted at least one executable bit set to indicate something was supposed to be executable. But it doesn't appear to be the case here, so `+x` is fine.

I do think this is more properly part of the prior commit introducing `/out`, since that change won't work without this one.

Reply:
Consolidated the `chmod` changes into 48e4ca2a895de6c54b4e7fbcf5321c28ddaf4074 and moved that before the other changes.